Tasks/Common/azure-arm-rest/package-lock.json

```diff
@@ -26,7 +26,7 @@
     "minimatch": {
       "version": "3.0.4",
       "resolved": "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz",
-      "integrity": "sha1-UWbihkV/AzBgZL5Ul+jbsMPTIIM=",
```

Comment:
this shouldn't be changed unless there is a change in package.json

Reply:
As discussed offline we should add this one.